nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] Let's kindly point out the class we are raising the error. It can be done like this:
`raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`